experiments/exp163_bert.py

```diff
@@ -0,0 +1,316 @@
+            ),
+            pip_dependency_groups=[
+                "--find-links https://storage.googleapis.com/libtpu-releases/index.html",
+                "--find-links https://storage.googleapis.com/libtpu-wheels/index.html",
+                "fasttext",
+                "datasets",
+                "filelock",
+                "torch",
+                "torch_xla[tpu]",
+                "accelerate",
+            ],
+        )
+
+        fasttext_consolidate_steps, bert_consolidate_steps = [], []
+        for keep_fraction in config.keep_fractions:
+            fasttext_consolidate_step = ExecutorStep(
+                name=f"documents/quality_filtering/{config.experiment_name}/fasttext/{input_data_source}",
+                fn=consolidate,
+                config=ConsolidateConfig(
+                    input_path=input_data_path,
+                    output_path=this_output_path(input_basename),
+                    filters=[
+                        FilterConfig(
+                            type=versioned("classify"),
+                            attribute_path=output_path_of(fasttext_inference, input_basename),
+                            name=versioned(f"{config.experiment_name}-fasttext_classifier"),
+                            label="__label__hq",
+                            threshold=versioned(None),
+                            keep_fraction=versioned(keep_fraction),
+                        ),
+                    ],
+                    ray_memory_limit_gb=12,
+                ),
+                pip_dependency_groups=["ddsketch"],
+            )
+
+            bert_consolidate_step = ExecutorStep(
+                name=f"documents/quality_filtering/{config.experiment_name}/bert/{input_data_source}",
+                fn=consolidate,
+                config=ConsolidateConfig(
+                    input_path=input_data_path,
+                    output_path=this_output_path(input_basename),
+                    filters=[
+                        FilterConfig(
+                            type=versioned("classify"),
+                            attribute_path=output_path_of(bert_inference, input_basename),
+                            name=versioned(f"{config.experiment_name}-bert_classifier"),
+                            label="hq",
+                            threshold=versioned(None),
+                            keep_fraction=versioned(keep_fraction),
+                        ),
+                    ],
+                    ray_memory_limit_gb=12,
+                ),
+                pip_dependency_groups=["ddsketch"],
+            )
+
+            fasttext_consolidate_steps.append(fasttext_consolidate_step)
+            bert_consolidate_steps.append(bert_consolidate_step)
+
+        steps.extend(fasttext_consolidate_steps)
+        steps.extend(bert_consolidate_steps)
+
+        # dolmino_dclm = get_dolmino_step_llama3("dclm")
+        for fasttext_consolidate_step, bert_consolidate_step in zip(
```

Comment:
I think this is cleaner to just have this in the same for loop above under keep_fraction so you don't have to do zip and worry about things being mis-aligned.